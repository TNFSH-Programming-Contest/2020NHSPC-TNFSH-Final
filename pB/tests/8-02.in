76
49.454 -108.817
95.448 -265.391
276.486 -292.791
440.889 -249.733
477.097 -89.0
430.125 11.549
330.309 55.586
156.12 48.736
121.87 -58.909
138.506 -151.875
192.328 -229.183
257.893 -241.905
365.538 -218.419
406.2 -148.6
392.938 -63.802
334.223 -20.744
240.279 -38.359
210.921 -85.331
234.407 -122.517
267.679 -149.918
305.844 -140.132
310.737 -104.903
287.0 -89.0
282.358 -124.474
246.15 -94.138
281.379 -50.102
357.709 -75.545
347.923 -174.382
228.536 -187.104
169.82 -124.474
192.328 -7.044
316.608 24.271
424.253 -42.273
440.889 -155.789
400.767 -237.012
251.043 -268.327
126.763 -237.99
79.79 -118.603
108.2 -29.4
67.069 19.378
129.698 16.442
157.099 82.007
190.371 75.157
332.266 82.986
435.996 52.65
479.054 -16.83
525.4 -89.0
506.454 -150.896
506.454 -264.412
386.088 -301.599
248.107 -312.363
168.842 -294.748
108.2 -327.4
57.283 -299.641
70.983 -272.241
18.139 -104.903
24.99 43.843
66.09 101.579
287.0 209.0
23.032 139.744
-11.0 -89.0
36.733 -245.819
48.476 -363.25
167.8 -327.4
318.566 -359.335
458.504 -303.556
498.626 -312.363
568.105 -262.455
534.833 -212.547
567.127 -124.474
456.546 85.922
246.15 104.515
140.463 99.622
126.763 46.778
45.54 39.928
60.219 -37.38
60.818 -319.354
