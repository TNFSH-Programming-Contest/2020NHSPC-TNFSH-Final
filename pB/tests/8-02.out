1037719.093